keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ApplicationEvent.kt

```diff
@@ -36,9 +35,9 @@ data class ApplicationActuationPaused(
   @JsonIgnore
   override val ignoreRepeatedInHistory = true
 
-  constructor(application: String, reason: String? = null, clock: Clock = Companion.clock) : this(
+  constructor(application: String, clock: Clock = Companion.clock) : this(
     application,
-    reason,
+    "Application $application paused",
```

Comment:
What's the role of the reason field?

The comment on line 28 describes the reason field as `The reason why actuation was paused`, but the string being passed `"Application $application paused"` doesn't seem to provide additional information.

Reply:
Yeah, fair point. I think I can just remove them.